app/index.js

```diff
@@ -10,7 +10,7 @@ import routes from './routes';
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import '../node_modules/toastr/build/toastr.min.css';
 import '../node_modules/griddle-react-bootstrap/dist/griddle-react-bootstrap.css';
-import '../ux/dev/css/styles.css';
+// import '../ux/dev/css/styles.css'; //ToDo: commenting MSUIX styles until fully integrated.
```

Comment:
Ok, I thought they were working fine and we were just trying to replace them.

That being the case, what do you think about removing the line altogether? We will need to make changes to this anyway since we'll remove bootstrap later on.